pinakes/data/email_template.html

```diff
@@ -0,0 +1,590 @@
+                        <table class="rh-data-table">
+                          <tbody>
+                            <tr>
+                              <th role="rowheader" scope="row">Ordered by:</th>
+                              <td>
+                                $requester_name<br/> (<a href="mailto:$orderer_email">$orderer_email</a>)
+                                <br/>
+                              </td>
+                            </tr>
+                            <tr>
+                              <th role="rowheader" scope="row">Order ID:</th>
+                              <td>$order_id</td>
+                            </tr>
+                            <tr>
+                              <th role="rowheader" scope="row">Order date:</th>
+                              <td>$order_date</td>
+                            </tr>
+                            <tr>
+                              <th role="rowheader" scope="row">Order time:</th>
+                              <td>$order_time</td>
+                            </tr>
+                            <tr>
+                              <th role="rowheader" scope="row">Product:</th>
+                              <td>$product</td>
+                            </tr>
+                            <tr>
+                              <th role="rowheader" scope="row">Portfolio:</th>
+                              <td>$portfolio</td>
+                            </tr>
+                            <tr>
+                              <th role="rowheader" scope="row">Platform:</th>
+                              <td>$platform</td>
+                            </tr>
+                          </tbody>
+                        </table>
+                      </td>
+                    </tr>
+                    <!-- end content block -->
+
+                    <!-- Content block -->
+                    <tr>
+                      <td class="rh-content__block">
+                        <strong><a href="$web_url/ui/catalog/approval/requests/detail/$approval_id" target="_blank">View more order details</a></strong> (requires authentication)
+                      </td>
+                    </tr>
+                    <!-- end content block -->
+
+                    <!-- Content block -->
+                    <tr>
+                      <td class="rh-content__block">
+                        <table align="center">
+                          <tr>
+                            <td class="rh-cta-link" align="center">
+                              <a target="_blank" href="$web_url/$approve_link">
+                                <span>
+                                  Process request
+                                </span>
+                              </a>
+                            </td>
+                          </tr>
+                        </table>
+                      </td>
+                    </tr>
+                    <!-- end content block -->
```

Comment:
This approval link doesn't exist yet, right? Maybe we should remove this for now.